server/datastore/redis/testing_utils.go

```diff
@@ -0,0 +1,60 @@
+package redis
+
+import (
+	"os"
+	"runtime"
+	"testing"
+	"time"
+
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	"github.com/gomodule/redigo/redis"
+	"github.com/stretchr/testify/require"
+)
+
+func SetupRedis(tb testing.TB, cluster, redir bool) fleet.RedisPool {
```

Comment:
I agree that those should be in test packages, I added it in a `testing_utils.go` as that is what was used in other packages like the mysql datastore. That being said, unless I missed something, I updated all callers, haven't run into a circular import issue.

Reply:
Right, because this is in the `redis` package still. If you were to create a new package called `redistest` somewhere, you'll end up with the circular import in the test that uses it because `NewRedisPool` is needed in both or something like that.

Reply:
Oh ok, gotcha. We'd probably need to move the `redis` package tests to `redis_test`. I'll take a shot at moving this when I'm done with the updated label timestamp.

Reply:
Updated to move `SetupRedis` to `redistest` package, and changing the `redis` package's tests that used it to blackbox tests in the `redis_test` package.